prow/plugins.yaml

```diff
@@ -167,3 +167,9 @@ external_plugins:
     events:
       - issue_comment
       - pull_request
+
+  maistra-test-tool:
```

Comment:
For completeness, please also add `maistra-test-tool` to the plugins list above.

Also, run `make gen` after making the changes.